openfisca_france/model/revenus/remplacement/indemnites_journalieres_securite_sociale.py

```diff
@@ -74,3 +73,9 @@ def function(self, simulation, period):
         )
 
         return period, result
+
+class date_arret_de_travail(Variable):
+    column = DateCol(default = nan)
```

Comment:
In short we have to choose between 2 ways of representing "undefined" dates.

## use not-a-time values

Numpy provides `numpy.datetime64('NaT')` which means not-a-time.

- pros: it's clean
- cons:
  - each formula has the responsibility to check for `NaT` values, which can easily be forgotten
  - `NaT` is not JSON-serializable so we have to change JSON serializer to handle this value by (simply) removing the key from the JSON dict

## use -infinite or +infinite dates

- pros:
  - no need to change JSON serialization
  - no need to change formulas
  - input variables have to choose if their default value is `MIN_DATE` or `MAX_DATE`, and the `DateCol(default = XXX)` is mandatory and constants are exported from `openfisca_core`
- cons:
  - not clean

--

My choice is -infinite and +infinite dates.